src/App/Fossa/ReleaseGroup/Common.hs

```diff
@@ -0,0 +1,31 @@
+module App.Fossa.ReleaseGroup.Common (
+  retrieveReleaseGroupId,
+  retrieveReleaseGroupRelease,
+) where
+
+import Control.Algebra (Has)
+import Control.Effect.Diagnostics (Diagnostics, errHelp, fatalText)
+import Data.Text (Text)
+import Fossa.API.Types (ReleaseGroup (..), ReleaseGroupRelease (..))
+
+retrieveReleaseGroupId :: Has Diagnostics sig m => Text -> [ReleaseGroup] -> m (Maybe Int)
+retrieveReleaseGroupId title releaseGroups = do
+  let filteredReleaseGroups = filter (\rg -> releaseGroupTitle rg == title) releaseGroups
+  case filteredReleaseGroups of
+    [] -> pure Nothing
+    [releaseGroup] -> pure . Just $ releaseGroupId releaseGroup
+    (_ : _ : _) ->
+      errHelp ("Navigate to the FOSSA web UI to rename your release groups so that they are unqiue" :: Text)
```

Comment:
```suggestion
      errHelp ("Navigate to the FOSSA web UI to rename your release groups so that they are unique." :: Text)
```

Reply:
Relatedly, as you find things that you think should be constrained in FOSSA's data-model to be unique, can you make tickets if you haven't yet?